<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="HMECL000695" name="com.automation.newati.regression.hometab.VerifyHomePageRegressionTest" tests="1" failures="0" timestamp="27 Mar 2019 05:19:31 GMT" time="269.294" errors="1">
  <testcase name="NewATI527_VerifyRecentActivitySection" time="269.294" classname="com.automation.newati.regression.hometab.VerifyHomePageRegressionTest">
    <error type="com.ascendlearning.automation.ui.exceptions.DriverException" message="Driver Verification exception:   Unable to fetch the Quiz Bank Assessment name">
      <![CDATA[com.ascendlearning.automation.ui.exceptions.DriverException: Driver Verification exception:   Unable to fetch the Quiz Bank Assessment name
	at com.automation.newati.pages.QuizPage.getQuizBankAssessmentName(QuizPage.java:728)
	at com.automation.newati.regression.hometab.VerifyHomePageRegressionTest.NewATI527_VerifyRecentActivitySection(VerifyHomePageRegressionTest.java:137)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.IndexOutOfBoundsException: Index: 2, Size: 1
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at com.automation.newati.pages.QuizPage.getQuizBankAssessmentName(QuizPage.java:724)
	... 25 more
]]>
    </error>
  </testcase> <!-- NewATI527_VerifyRecentActivitySection -->
</testsuite> <!-- com.automation.newati.regression.hometab.VerifyHomePageRegressionTest -->
